### ISSUE TITLE
Duda BCNF

### ISSUE THREAD

**Issue by chocolito24** [IIC2413/Syllabus-2023-1#106]
Al descomponer las tablas a BCNF, si existen dependencias desde un atributo que no es superllave, debemos descomponer la tabla. Si tenemos por ejemplo, la tabla (rut, n°telefono, genero, edad), como se separaría esta tabla? Ya que por ejemplo, si separamos en una tabla (rut, n°telefono), tenemos que el rut sería la superllave, pero rut -> n°telefono, y n°telefono -> rut, lo que iría en contra de la definición de BCNF o no? 
Muchas gracias!

**Comment by NicolasOlmosQuiroga** [comment]
Claro, al descomponer la tabla (rut, n°telefono, genero, edad) a BCNF, con lo tu ejemplo tendriamos que:

rut -> n°telefono
n°telefono -> rut

Ambas dependencias son funcionales, entonces la tabla no cumple con BCNF.

Una posible solución sería separar las tablas de la siguiente manera:

Tabla 1: (rut, n°telefono)

Superclave: {rut, n°telefono}

No existen dependencias funcionales en esta tabla, por lo que cumple con BCNF.

Tabla 2: (rut, genero, edad)

Superclave: {rut}

No existen dependencias funcionales en esta tabla, por lo que cumple con BCNF.

Saludos.
